impl/src/test/java/io/kroxylicious/testing/kafka/common/KeytoolCertificateGeneratorTest.java

```diff
@@ -40,6 +41,30 @@ public void generatesKeyStore() throws Exception {
         assertThat(ks.getType()).isEqualTo(generator.getKeyStoreType());
     }
 
+    @Test
+    public void generatesKeyStoreWithIPDomain() throws Exception {
+        var generator = new KeytoolCertificateGenerator();
+        generator.generateSelfSignedCertificateEntry("test@kroxylicious.io", "127.0.0.1", "Dev",
+                "Kroxylicious.io", null, null, "US");
+
+        var keystore = generator.getKeyStoreLocation();
+        assertThat(keystore).isNotEmpty();
+        var keystoreFile = new File(keystore);
+        assertThat(keystoreFile).exists();
+        var password = generator.getPassword();
+
+        var ks = KeyStore.getInstance(keystoreFile, password.toCharArray());
+        var aliases = aliasList(ks);
```

Comment:
Ideally, tests should test one thing.  This test is all about SAN IP address - so just test that aspect.